new quickstart guides (#3060)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/checkfortoken.md

> Check for an existing session in 2 steps. First, initialize the authorization state mananger, `authStateManager`, from a saved session. Then check if the access token is still valid by checking the expiry. The state manager session and it's associated token are already saved by the `signIn` function calling `writeToSecureStorage`.

Reviewer: ```suggestion
Check for an existing session in two steps. First, initialize the authorization state manager (`authStateManager`) from a saved session. Then, verify that the access token is still valid by checking the expiry. The state manager session and its associated token are already saved by the `signIn` function calling `writeToSecureStorage`.
```

Update the `tokenExpired` utility function to check for token expiry:

```swift
func configureSDK() {
...
  // Check for an existing session
  self.authStateManager = OktaOidcStateManager.readFromSecureStorage(for: config)
  if let authStateManager = self.authStateManager,
    let tokenString = authStateManager.accessToken,
    !tokenExpired(tokenString) {
      updateStatus("Signed In.", infoText: "", signedInStatus: true)
    }
}
```

> Then add this code to the end of `configureSDK` to initialize the state manager from a saved session, and to check if the token is still valid:

Reviewer: ```suggestion
Then add this code to the end of `configureSDK` to initialize the state manager from a saved session and to check if the token is still valid:
```

Check for an existing session in two steps.

1. Initialize the authorization state manager (`authStateManager`) from a saved session.
2. Verify that the access token is still valid by checking the expiry.

The state manager session and its associated token are already saved by the `signIn` function calling `writeToSecureStorage`.

> Update the `tokenExpired` utility function to check for token expiry:

Reviewer 2: Clarity and succinctness:

Check for an expired token by updating the `tokenExpired` utility function:

Author: updated

```swift
func tokenExpired(_ tokenString: String?) -> Bool {
  guard let accessToken = tokenString,
  let tokenInfo = OKTIDToken.init(idTokenString: accessToken) else {
    return false
  }
    
  return Date() > tokenInfo.expiresAt
}
```

> Then add this code to the end of `configureSDK` to initialize the state manager from a saved session and to check if the token is still valid:

Reviewer 2: I think the following is a bit clearer and more succinct:

Then initialize the state manager from a saved session if the token is valid by adding this code to the end of `configureSDK`.

Author: Agreed. Updated.

```swift
func configureSDK () {

> ...

Reviewer 3: ```suggestion
   ...
```
Indent to match the method body.

Author: updated
